packages/web-worker/package.json

```diff
@@ -28,11 +34,8 @@
     ],
     "scripts": {
         "clean": "rm -fr dist && mkdir dist",
-        "build": "npm run clean && npm run build:dist && npm run build:dist:min",
-        "build:dist": "webpack",
-        "build:dist:min": "webpack --optimization-minimize --output-filename=bugsnag.web-worker.min.js",
-        "build:dist:esm": "webpack --config esm.config.js",
-        "build:dist:esm.min": "webpack --config esm.config.js --optimization-minimize --output-filename=bugsnag.web-worker.min.mjs",
```

Comment:
these don't appear to have been used. it looks like we're also publishing this to cdn but then there are no docs for that. so do we need to support cdn for web workers?

Reply:
I think that's one for @tomlongridge - as far I understand the CDN was a requirement, but if it's not been documented we will need to remedy that

Reply:
It sounds like it, but also opens the question about whether we have tests to cover that scenario. I'd suggest ticketing for future, but keep doing what we're doing now as far as publishing is concerned so we don't stop publishing if anyone is using it?